lms/templates/admin/organizations.html.jinja2

```diff
@@ -35,27 +35,35 @@ Organizations
         <div class="container">
             <div class="table-container">
                 <table class="table is-fullwidth">
-                  <thead>
-                    <tr>
-                      <th></th>
-                      <th>Public ID</th>
-                      <th>ID</th>
-                      <th>Name</th>
-                    </tr>
-                  </thead>
-
-                  <tbody>
-                    {% for org in organizations %}
+                    <thead>
                         <tr>
-                          <td>
-                            <a class="button" href="{{ request.route_url('admin.organization', id_=org.id) }}">View</a>
-                          </td>
-                          <td>{{org.public_id}}</td>
-                          <td>{{org.id}}</td>
-                          <td>{{org.name}}</td>
-                       </tr>
-                    {% endfor %}
-                  </tbody>
+                          <th></th>
+                          <th>Public ID</th>
+                          <th>ID</th>
+                          <th>Name</th>
+                          <th>Root</th>
+                        </tr>
+                    </thead>
```

Comment:
The indentation in this file was mixed up. There was both 2 and 4 space. I've moved it to 4 space as my IDE was doing my head in.